plugins/virtualenv/plugin.zsh

```diff
@@ -1,15 +1,34 @@
 # Color definitions
-GEOMETRY_COLOR_VIRTUALENV=${GEOMETRY_COLOR_PROMPT:-green}
+DEFAULT_COLOR=${GEOMETRY_COLOR_PROMPT:-green}
+GEOMETRY_COLOR_VIRTUALENV=${GEOMETRY_COLOR_VIRTUALENV:-$DEFAULT_COLOR}
+GEOMETRY_COLOR_CONDA=${GEOMETRY_COLOR_CONDA:-$DEFAULT_COLOR}
+GEOMETRY_VIRTUALENV_CONDA_SEPARATOR=${GEOMETRY_VIRTUALENV_CONDA_SEPARATOR:-:}
+
 
 geometry_prompt_virtualenv_setup() {}
 
 geometry_prompt_virtualenv_check() {
-  test $VIRTUAL_ENV || return 1
+    [ -n "${VIRTUAL_ENV}" -o -n "${CONDA_PREFIX}" ]
 }
 
 geometry_prompt_virtualenv_render() {
-  local ref=$(basename $VIRTUAL_ENV)
-  echo "$(prompt_geometry_colorize $GEOMETRY_COLOR_VIRTUALENV "(${ref})")"
+
+    if [ -n "${VIRTUAL_ENV}" ]; then
+
+        virtualenv_ref=$(basename $VIRTUAL_ENV)
+        virtualenv_str="$(prompt_geometry_colorize $GEOMETRY_COLOR_VIRTUALENV ${virtualenv_ref})"
+
+        if [ -n "${CONDA_PREFIX}" ]; then
```

Comment:
It looks great and works perfectly. I'm not too fan of duplicating the `CONDA` block though. I'd create 3 separate block for:

1- Generate virtualenv prompt (`if VIRTUAL_ENV`)
2- Generate separator if there's conda AND virtualenv are available (`if VIRTUAL_ENV && CONDA_PREFIX`)
3- Generate conda prompt (`if CONDA_PREFIX`)

Anyways it's a nitpick.